microceph/cmd/microceph/pool.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/canonical/microcluster/microcluster"
+	"github.com/spf13/cobra"
+
+	"github.com/canonical/microceph/microceph/api/types"
+	"github.com/canonical/microceph/microceph/client"
+)
+
+type cmdPool struct {
```

Comment:
This would also be required to be registered under `cmd/microceph/main.go` to be associated as a sub-command to the `microceph` command.

Reply:
Done.